ktor-client/ktor-client-winhttp/windows/src/io/ktor/client/engine/winhttp/WinHttpClientEngineConfig.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.engine.winhttp
+
+import io.ktor.client.engine.*
+import io.ktor.http.*
+
+public class WinHttpClientEngineConfig : HttpClientEngineConfig() {
+
+    /**
+     * A value indicating whether HTTP 2.0 protocol is enabled in WinHTTP.
+     * The default value is true.
+     */
+    public var protocolVersion: HttpProtocolVersion = HttpProtocolVersion.HTTP_2_0
+
+    /**
+     * A value that allows to set required security protocol versions.
+     * By default will be used system setting.
+     */
+    public var securityProtocols: WinHttpSecurityProtocol = WinHttpSecurityProtocol.Default
+
+    /**
+     * A value that disables TLS verification for outgoing requests.
+     */
+    public var sslVerify: Boolean = true
```

Comment:
It was used only for tests to check connection to web sites with self signed certificates. Could we expose it for end users?

Reply:
Why not? [CURL exposes `sslVerify` too.](https://api.ktor.io/ktor-client/ktor-client-curl/io.ktor.client.engine.curl/-curl-client-engine-config/ssl-verify.html) Users may use it for self-host services.